app/build.gradle

```diff
@@ -44,4 +44,5 @@ dependencies {
     implementation 'com.android.support:design:28.0.0'
     implementation 'com.android.support:preference-v7:28.0.0'
     implementation 'com.android.support:recyclerview-v7:28.0.0'
+    implementation 'com.afollestad:sectioned-recyclerview:0.5.0'
```

Comment:
I'm not too happy about this dependency, since it seems to be pretty much dead. The author even removed it from his GitHub profile

Reply:
I know, this is totally outdated, but also used in NC Files App…
If you find a proper replace, I am more than happy to change it.
(also the other way around, if I find one for NC Files App).
But for now it "just works" (for >500k users…)